### PR TITLE
Create private-dns.md to fix Azure/azure-sdk-for-python#24212

### PR DESCRIPTION
**Pull request by navba-MSFT**
Fixes #Azure/azure-sdk-for-python#24212

More details:
Content of this page is empty: https://docs.microsoft.com/en-us/python/api/overview/azure/privatedns?view=azure-python

So this private-dns.md includes the content and helps to fix this issue.

### CODE REVIEW

File: docs-ref-services/latest/private-dns.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+To get started with Azure Private DNS, see [Create an Azure private DNS zone using the Azure portal](/azure/dns/private-dns-getstarted-portal).
+
+## Management API
+
+Create, configure and manage and scale private dns in Azure from your code with the management API.
+
+Install the library via pip.
+
+```bash
+pip install azure.mgmt.privatedns
```
Comment (msyyc) [inline docs-ref-services/latest/private-dns.md:16]:
```suggestion
pip install azure-mgmt-privatedns
```

File: docs-ref-services/latest/private-dns.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+> [!div class="nextstepaction"]
+> [Explore the Management APIs](/python/api/azure-mgmt-privatedns/azure.mgmt.privatedns)
+
```
Comment (msyyc) [inline docs-ref-services/latest/private-dns.md:21]: I don't know the effect of line19-20. Is it expected?
image

Comment (navba-MSFT) [reply docs-ref-services/latest/private-dns.md:21]: @msyyc Yes it is required as we need a button to help users navigate to the Management API. 
Sharing a sample screenshot if that helps.

image

**Review by scbedd — APPROVED** [review]
I have updated the links to remove warnings on absolute, and duplicated the content into the preview directory as well. Next CI run will ensure this populates.

**Pull request merged**